config/build.jam

```diff
@@ -0,0 +1,14 @@
+# Copyright René Ferdinand Rivera Morell
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+import-search /boost/predef/tools ;
+import predef ;
+
+project
+    : requirements
+        # Set these to computed values according to the build.
+        [ predef.address-model ]
+        [ predef.architecture ]
+    ;
```

Comment:
What is the purpose of this file? Is it actually needed?

Reply:
I thought you said these properties are auto-detected and always set now, are they not?

Config projects are used from the main build Jamfile, are these (and all other) properties not forwarded to the config projects when they are compiled? I think, they need to be.